KsApi/models/Category.swift

```diff
@@ -87,15 +87,57 @@ extension Category {
     case id, name, parentId, _parent = "parentCategory", subcategories, totalProjectCount
   }
 
+  private enum v1CodingKeys: String, CodingKey {
+    case parentId = "parent_id"
+    case parentName = "parent_name"
+  }
+
   public init(from decoder: Decoder) throws {
     let values = try decoder.container(keyedBy: CodingKeys.self)
-    self.id = try values.decode(String.self, forKey: .id)
+
+    self.id = try stringOrInt(in: values, for: .id)
     self.name = try values.decode(String.self, forKey: .name)
-    self.parentId = try? values.decode(String.self, forKey: .parentId)
-    self._parent = try? values.decode(ParentCategory.self, forKey: ._parent)
+    self.parentId = try? Category.tryParentId(decoder)
+    self._parent = try? Category.tryParentCategory(decoder)
     self.subcategories = try? values.decode(SubcategoryConnection.self, forKey: .subcategories)
     self.totalProjectCount = try? values.decode(Int.self, forKey: .totalProjectCount)
   }
+
+  /* Tries to decode the parent category from the expected GraphQL structure
```

Comment:
Not sure how I'm feeling about this. It seems to work but I'm going to take another look tomorrow at what would be involved to just split these two models out completely.

Reply:
Ok, started down a different path and don't really feel like using separate models would be significantly better. Let me know what you think @Scollaco @ifbarrera.